install-instructions.php

```diff
@@ -249,8 +249,7 @@
 									<a href="https://owncloud.com/download/?ref=orgB" target="_blank" class="button orange primary mb30">Test Enterprise Edition free for 30 days</a>
 									<hr />
 									<div class="download-notes">
- 										<p><strong>NOTE</strong>: The appliances have been updated to the <a href="https://owncloud.org/blog/new-in-owncloud-9-1/">freshly released ownCloud 9.1</a>. </p>
-										<p><strong>NOTE</strong>: The appliances have been updated to the <a href="https://owncloud.org/blog/owncloud-server-9-0-released/">freshly released ownCloud 9.0</a> but have only gone through absolutely minimal testing. It is recommended to not use these for important data yet - giving them a trial run however and giving us feedback is greatly appreciated!</p>
+ 										<p><strong>NOTE</strong>: The appliances have been updated to <a href="https://owncloud.org/blog/introducing-owncloud-x/">ownCloud 10.0</a> but have only gone through absolutely minimal testing. It is recommended to not use these for important data yet - giving them a trial run however and giving us feedback is greatly appreciated!</p>
```

Comment:
My item 1, about the owncloud/owncloud username is in the install doc, so good.
Item 2, the https problem accessing the activation page - I made PR https://github.com/owncloud/documentation/pull/3160 to add that little gem to the install doc.

It would be nice to have a link here on the Appliance download "page" to the appliance install doc at https://doc.owncloud.com/server/10.0/admin_manual/enterprise/appliance/installation.html#download-the-appliance - if that had been there then I would have gone to the doc in the first place.